src/app/data-model-provider/Provider.h

```diff
@@ -79,6 +79,14 @@ class Provider : public ProviderMetadataTree
     ///    - validation of ACL/timed interaction flags/writability, if those checks are desired.
     virtual ActionReturnStatus WriteAttribute(const WriteAttributeRequest & request, AttributeValueDecoder & decoder) = 0;
 
+    ///   Indicates the start/end of a series of list operations. This function will be called either before the first
+    ///   Write operation or after the last one of a series of consequence attribute data of the same attribute.
+    ///
+    ///   1) This function will be called if the client tries to set a nullable list attribute to null.
```

Comment:
Once or twice?

Reply:
This seems a mistake from my side, it was supposed to be "once" per write begin and once per write end. So for a whole write operation would be two calls.